htdocs/product/class/product.class.php

```diff
@@ -769,6 +769,12 @@ class Product extends CommonObject
 	 */
 	public $mandatory_period;
 
+	/**
+	 * 0=This service or product is not managed in stock, 1=This service or product is managed in stock
+	 *
+	 * @var boolean
+	 */
+	public $stockable_product = true;
```

Comment:
Using boolean must be avoid as it can't be used to evoluate.
It is better to use int so
/* @var integer */
public $stockable_product = 1;

Reply:
ok